examples/three_layer_constant_fluxes.jl

```diff
@@ -392,62 +392,78 @@ tke_budget_statistics = turbulent_kinetic_energy_budget(model, b=b, p=p, U=U, V=
 fields_to_output = merge(model.velocities, model.tracers, (e=e, ϵ=dissipation))
 
 statistics_to_output = merge(primitive_statistics, subfilter_flux_statistics, tke_budget_statistics)
-
-function init_save_some_metadata!(file, model)
-    file["parameters/coriolis_parameter"] = f
-    file["parameters/density"] = 1027.0
-    file["boundary_conditions/θ_top"] = Qᶿ
-    file["boundary_conditions/θ_bottom"] = dθdz_deep
-    file["boundary_conditions/u_top"] = Qᵘ
-    file["boundary_conditions/u_bottom"] = 0.0
-    return nothing
-end
+statistics_to_output = Dict(string(k) => v for (k, v) in statistics_to_output)
 
 @info "Garnishing output writers..."
 
+ρ₀ = 1027
+cₚ = 4000
+
+global_attributes = (
+    name = name,
+    thermocline_type = thermocline_type,
+    reference_density = ρ₀,
+    specific_heat_capacity = cₚ,
+    buoyancy_flux = Qᵇ,
+    momentum_flux = Qᵘ,
+    heat_flux = Qᶿ,
+    heat_flux_Wm⁻² = - ρ₀ * cₚ * Qᶿ,
+    coriolis_parameter = f,
+    thermal_expansion_coefficient = α,
+    gravitational_acceleration = g,
+    boundary_condition_θ_top = Qᶿ,
+    boundary_condition_θ_bottom = dθdz_deep,
+    boundary_condition_u_top = Qᵘ,
+    boundary_condition_u_bottom = 0.0,
```

Comment:
How can we automate saving parameters of the boundary conditions?

Reply:
Was thinking this could be part of https://github.com/CliMA/Oceananigans.jl/issues/1313 (also see comment below)